commandhandlers.go

```diff
@@ -141,9 +153,10 @@ var commandsGlobal = []Command{
 	},
 	{
 		Data: api.CreateCommandData{
-			Name:        "config",
-			Description: "Configure the verification channel and role",
-			Type:        discord.ChatInputCommand,
+			Name:                     "config",
+			Description:              "Configure the verification channel and role",
+			Type:                     discord.ChatInputCommand,
+			DefaultMemberPermissions: permissionRequireAdministrator(),
```

Comment:
Is it possible to inline this? I'd assume it's possible to write just `&discord.PermissionAdministrator`, which would get the point across more clearly.

Reply:
`&discord.PermissionAdministrator` doesn't seem to work even though it seems obvious it should. Here's a [stackoverflow post](https://stackoverflow.com/a/35146856) I found that made me end up with this workaround.

Reply:
Wow, that's annoying as hell. This is yucky but we can allow it.

Another potential fix I found in that stackoverflow thread was this:

```go
func ConstRef[E any](e E) *E {
    return &e
}
```

Up to you. I'll approve this and you can make the change if you think it's worth it.